src/ligero/data_structures.rs

```diff
@@ -182,3 +182,6 @@ where
 
     pub(crate) columns: Vec<Vec<F>>,
 }
+
+/// The Proof type for Ligero, which amounts to an array of individual ligero proofs
+pub type LigeroPCProofArray<F, C> = Vec<LigeroPCProof<F, C>>;
```

Comment:
You might be missing formatting, typically `cargo fmt` will add an empty line to the end of file